velox/common/caching/SsdFileTracker.h

```diff
@@ -59,14 +62,31 @@ class SsdFileTracker {
       const std::vector<int32_t>& regionPins);
 
   // Expose the region access data. Used in checkpointing cache state.
-  std::vector<int64_t>& regionScores() {
+  std::vector<asan_atomic<uint64_t>>& regionScores() {
     return regionScores_;
   }
 
+  void setRegionScores(const std::vector<int64_t>& scores) {
+    VELOX_CHECK_EQ(scores.size(), regionScores_.size());
+    for (auto i = 0; i < scores.size(); ++i) {
+      regionScores_[i] = scores[i];
+    }
+  }
+
+  /// Exports a copy of the scores. Asan will report error if the
+  /// scores are not copied before being e.g. written to a file.
```

Comment:
What do you mean by "cores are not copied before being e.g. written to a file." here?